lib/kafka-consumer.js

```diff
@@ -585,7 +631,7 @@ KafkaConsumer.prototype._consumeNum = function(timeoutMs, numMessages, cb) {
  * @private
  * @see consume
  */
-KafkaConsumer.prototype._consumeNumOfPartition = function(timeoutMs, numMessages, topic, partition, cb) {
+KafkaConsumer.prototype._consumeNumOfPartition = function(timeoutMs, numMessages, topic, partition, cb, onlyApplyTimeoutToFirstMessage = false) {
```

Comment:
Why does the last parameter get a default?

Reply:
Just to be on the safe side. But we can omit it. I've removed it.